coalib/bearlib/aspects/meta.py

```
@@ -72,3 +75,24 @@ class Sub(subcls, aspectbase, metaclass=aspectclass):
 
     def __repr__(cls):
         return '<%s %s>' % (type(cls).__name__, repr(cls.__qualname__))
+
+
+def issubaspect(aspect, subaspect):
+    """
+    This function checks whether or not ``subcls`` is a subaspect of cls
+    """
+    if not isinstance(aspect, aspectclass):
+        logging.warning(
```

Won't it be better if we raise `TypeError` instead of `logging.warning`?